app/assets/stylesheets/search.scss

```diff
@@ -0,0 +1,268 @@
+//to be removed
```

Comment:
?

Reply:
Some styles have been added from the previous PRs; since those PRs were yet to be merged and I required them here. Will be removed in the final code cleanup

Reply:
@tachyons